app/ports/models/stats.py

```diff
@@ -39,9 +42,12 @@ class Meta:
             models.Index(fields=['user', '-timestamp']),
             models.Index(fields=['user', 'timestamp']),
             models.Index(fields=['os_version']),
+            models.Index(fields=['os_version', 'os_arch']),
```

Comment:
Doesn't this need to be `os_version, build_arch`?

Reply:
Thinking a bit further. Will we need to index on `['platform', 'os_version', 'build_arch']`? As soon as someone submits statistics from Linux, we might need to adapt the code in the queries, and then we'll probably need to check all three rather than just two..

Reply:
Thank you. We will need an index on all three: `['platform', 'os_version', 'build_arch']`.

Reply:
And also `['os_version', 'build_arch', 'cxx_stdlib']`.